Build.cs

```diff
@@ -412,6 +417,7 @@ private bool ValidateROM(string FileName)
 
         private void MakeROM(string InFile, string FileName, BackgroundWorker worker)
         {
+            if(!cLogOnly.Checked)
```

Comment:
Please use curly brackets. This is too confusing to read and looks like a mistake.